File: app/Author.php
<?php namespace App;

use Illuminate\Database\Eloquent\Model;

class Author extends Model {

    protected $table = 'authors';

    protected $fillable = [
        'id',
        'name',
        'title',
        'url',
        'image',
        'biography',
        'media-links'
    ];

    protected $visible = [
        'id',
        'name',
        'title',
        'url',
        'image',
        'biography',
        'media-links'
    ];

    public function articles()
    {
        return $this->hasMany('App\Article');
    }

    public function crawl()
    {
        return $this->morphOne('App\Crawl', 'crawlable');
    }

}

File: krautreporter/entity/author.go
package entity

import "time"

type Author struct {
	ID          int
	Ordering    int
	Name        string
	Title       string
	URL         string
	Biography   string
	SocialMedia string
	CreatedAt   time.Time
	UpdatedAt   time.Time

	Images []Image `gorm:"polymorphic:Imageable;"`
	Crawl  Crawl   `gorm:"polymorphic:Crawlable;"`
}
